docs/flow-setup-tutorial.md

```diff
@@ -0,0 +1,34 @@
+# Flow setup Tutotrial
+
+Flow is a static-type checker for Javascript code
+
+## The steps to install Flow correctly from scratch are
+
+####  Installation tip : install flow in the root folder of zulip-mobile
```

Comment:
That is not desirable. We want it either installed globally, or via `package.json` in `./node_packages`.

Reply:
I found this line "Flow works best when installed per-project with explicit versioning rather than globally." in the flow's documentation , that's why i added that line.

Reply:
Ok, that is reasonable. Then we can make it run locally. We don't need to add instructions but just implement it.

Reply:
We are currently installing the correct Flow version. The only thing a user needs to do is `npm run test:flow`

Reply:
i have written this guide for yarn users , should i update for npm users as well ?